### PR TITLE
add readme, workflows, codeowners and fix lint warnings

### PR DESCRIPTION
**Pull request by mpetrunic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @sweetpea22 @web3/web3-js
```
Comment (jdevcs) [inline .github/CODEOWNERS:1]:
```suggestion
* @web3/web3-js
```
@sweetpea22  is not in our org?

**Comment by jdevcs** [comment]
lgtm except there is a question above. Thanks @mpetrunic

File: .github/CODEOWNERS (hunk 1 above)
Comment (mpetrunic) [reply .github/CODEOWNERS:1]: That's Cindy, added her as outside collaborator on this repo^^

**Review by jdevcs — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Comment by jdevcs** [comment]
For future PRs this repo should also be protected with signed commits I think, via branch protection? @mpetrunic

**Comment by mpetrunic** [comment]
> For future PRs this repo should also be protected with signed commits I think, via branch protection? @mpetrunic

I'm fine with that, but this is your decision to enforce it :)

**Pull request merged**